docs/docs/architecture/features-limitations.md

```diff
@@ -4,19 +4,19 @@ This section lists planned features and current limitations of Contrast.
 
 ## Availability
 
-- **Platform support**: At present, Contrast is exclusively available on Azure AKS, supported by the [Confidential Container preview for AKS](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers-on-aks-preview). Expansion to other cloud platforms is planned, pending the availability of necessary infrastructure enhancements.
+- **Cloud platform support**: At present, Contrast is exclusively available on Azure AKS, supported by the [Confidential Container preview for AKS](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers-on-aks-preview). Expansion to other cloud platforms is planned, pending the availability of necessary infrastructure enhancements.
 - **Bare-metal support**: Support for running [Contrast on bare-metal Kubernetes](../howto/cluster-setup/bare-metal.md) is available for AMD SEV-SNP and Intel TDX.
 
 ## Kubernetes features
 
 - **Persistent volumes**: Contrast only supports volumes with [`volumeMode: Block`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#volume-mode). These block devices are provided by the untrusted environment and should be treated accordingly. The [transparent encryption feature](architecture/secrets.md#secure-persistence) is recommended for secure persistence.
 - **Port forwarding**: This feature [isn't yet supported by Kata Containers](https://github.com/kata-containers/kata-containers/issues/1693). You can [deploy a port-forwarder](https://docs.edgeless.systems/contrast/deployment#connect-to-the-contrast-coordinator) as a workaround.
 - **Resource limits**: Contrast doesn't support setting CPU limits on bare metal. Adding a resource request for CPUs will lead to attestation failures.
-- **Image pull secrets**: registry authentication is only supported on AKS, see [Registry Authentication](howto/registry-authentication.md#bare-metal) for details.
+- **Image pull secrets**: registry authentication is only supported on AKS, see [Registry Authentication](../howto/registry-authentication.md#bare-metal) for details.
```

Comment:
We should probably have a link checker.

Reply:
Actually, I'd expect docusaurus to check this, and it does in many cases. Unsure why this one went through. Maybe because it wasn't clear it's a local path.